packages/flame-css/README.md

```diff
@@ -0,0 +1,36 @@
+# Flame css
+
+`flame-css` is an alternative offering to `flame`.
+
+It offers a sass codebase and an easy to consume css file.
+
+@TODO:
+
+- [ ] Hook flame-tokens/flame-foundations as primary variables
+- [ ] Integrate tailwind to output utility classes
```

Comment:
Let's rephrase this to avoid implementation details. Let's instead mention that we'd like to include a guide on how to integrate [utility-first CSS](https://css-tricks.com/in-defense-of-utility-first-css/) on top of Flame CSS.